dradis/dradis-ce
# Issue states crud

### Spec
Implement CRUD for issue states

**Proposed solution**
1. Add state column to issues
2. Update issues CRUD to handle states
3. Update dradis-projects to handle states for export/import
4. Update dradis-plugins to handle states for plugin imports

### How to test

**Test CRUD**

1. Create an issue and choose a state for it
2. Confirm that the issue was created with the state
3. Update the issue state
4. Confirm that the issue was updated with the state

**Test Export/Import**

1. Add multiple issues in your project
2. Export the project
3. Open the XML file and confirm that it contains issue states
4. Import the XML in another project
5. Confirm that the issue’s state was imported correctly

**Test Plugin Upload**

1. Go to the uploads page
2. Select a plugin
3. Select a state
4. Upload a sample plugin file
5. Confirm that the created issues have the selected state
